nvm.sh

```diff
@@ -1231,6 +1231,25 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ -z "$NVM_MAKE_JOBS" ]; then
+    if [ "_$NVM_OS" = "_linux" ]; then
+      NVM_CPU_THREADS="$(grep -c 'core id' /proc/cpuinfo)"
+    elif [ "_$NVM_OS" = "_freebsd" ] || [ "_$NVM_OS" = "_darwin" ]; then
+      NVM_CPU_THREADS="$(sysctl -n hw.ncpu)"
+    elif [ "_$NVM_OS" = "_sunos" ]; then
+      NVM_CPU_THREADS="$(psrinfo | wc -l)"
+    fi
+    local NVM_CPU_THREAD_VALID
+    NVM_CPU_THREAD_VALID=$(nvm_is_natural_num $NVM_CPU_THREADS)
+    if [ -z "$NVM_CPU_THREADS" ] || [ "$NVM_CPU_THREAD_VALID" != "true" ] ; then
+      echo "Can not determine how many thread(s) we can use, set to only 1 now." 1>&2
+      echo "Please report an issue on GitHub to help us make it better and run it faster on your computer!" 1>&2
+      NVM_MAKE_JOBS="1"
+    else
+      NVM_MAKE_JOBS=$(($NVM_CPU_THREADS - 1))
```

Comment:
can we output something to stdout here indicating what the number of threads has been automatically set to?